### PR TITLE
Styling L5kit documentation

### PR DESCRIPTION
**Pull request by perone**
This PR will change the theme of our Sphinx docs and fix some title issues where there were `None` being displayed due to the lack of title in some toctrees.

## Screenshot 1
image

## Screenshot 2
image

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -55,12 +55,20 @@
...
+      ("Github", "https://github.com/lyft/l5kit"),
+  ]
+}
 
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = []
+html_static_path = ["./css"]
+html_css_files = ["custom.css"]
```
Comment (lucabergamini) [inline docs/conf.py:71]: that seems empty

File: l5kit/Pipfile [hunk 2]
```diff
@@ -36,3 +36,5 @@ torch = ">=1.5.0,<=1.7.1"
 torchvision = ">=0.6.0,<=0.8.2"
 shapely = "*"
 bokeh = "*"
+sphinx-press-theme = "*"
```
Comment (lucabergamini) [inline l5kit/Pipfile:39]: can we remove the old one (line 13)?

File: l5kit/setup.py [hunk 3]
```diff
@@ -7,8 +7,8 @@
 setup(
     name="l5kit",
     version=__version__,
-    description="Lyft Autonomous Vehicle Research library",
-    author="Lyft Level 5",
+    description="Level-5 Autonomous Vehicle Research library",
+    author="Level 5",
     author_email="l5kit@lyft.com",
```
Comment (lucabergamini) [inline l5kit/setup.py:12]: wondering if this has changed too

File: l5kit/setup.py [hunk 4]
```diff
@@ -37,7 +37,7 @@
     extras_require={
         "dev": ["pytest", "mypy", "types-PyYAML", "setuptools", "twine", "wheel", "pytest-cov",
                 "flake8", "isort", "Sphinx", "sphinx-rtd-theme", "recommonmark",
```
Comment (lucabergamini) [inline l5kit/setup.py:39]: can we remove `sphinx-rtd-theme`?

**Review by lucabergamini — APPROVED** [review]
just minors, but LGTM

File: l5kit/Pipfile (hunk 2 above)
Comment (perone) [reply l5kit/Pipfile:38]: Removed

File: l5kit/setup.py (hunk 4 above)
Comment (perone) [reply l5kit/setup.py]: Removed

File: l5kit/setup.py (hunk 3 above)
Comment (perone) [reply l5kit/setup.py:12]: Good question, for the moment I think they are still redirecting, I will double-check that when we change the repo org.

File: docs/conf.py (hunk 1 above)
Comment (perone) [reply docs/conf.py:71]: This one is intentional in case we want to customize the template later.

**Comment by perone** [comment]
@lucabergamini @theDebugger811 will need another review as it dismissed the previous one.

**Review by theDebugger811 — APPROVED** [review]
Looks good. Thanks @perone

**Pull request merged**